templates/login.html

```diff
@@ -0,0 +1,36 @@
+{{ define "content" }}
+
+<h2 class="auth-main__title">
+  Login to your account
+</h2>
+
+{{ if .notices }}
+  <div class="alert alert--success">
+    {{ range $notice := .notices }}
+      {{ $notice }}
+    {{ end }}
+  </div>
+{{ end }}
+
+{{ if .errors }}
+  <div class="alert alert--danger">
+    {{ .errors }}
+  </div>
+{{ end }}
```

Comment:
For all flashes. how about defining a partial in the Goview Config

```go
return goview.Config{
  Root:      "templates",
  Extension: ".html",
  Master:    "layouts/authentication",
  Partials:  []string{"partials/flashes"}
}
```

Then use in the view as:

```go
{{template "flashes" .}}
```

You can find a detailed example on [the Goview repository](https://github.com/foolin/goview/tree/master/_examples/gin-multiple).

Reply:
I don't know why I got 500 when I use Partials config in the goview. I did this [way](fcca86412e5f03b6f3270caf678ea5a0d74df606) instead. What do you think?